aws-es-proxy.go

```diff
@@ -253,6 +253,14 @@ func addHeaders(src, dest http.Header) {
 	if val, ok := src["Content-Type"]; ok {
 		dest.Add("Content-Type", val[0])
 	}
+
```

Comment:
Might be worth correspondingly updating the comment to this function, aside from that LGTM 👍🏽.